.github/workflows/e2e_tests.yml

```diff
@@ -41,61 +27,23 @@ jobs:
 
-      - name: Cache Playwright browsers
-        id: playwright-cache
-        uses: actions/cache@v4
-        with:
-          path: ~/.cache/ms-playwright
-          key: playwright-${{ runner.os }}-${{ hashFiles('**/pnpm-lock.yaml', '**/playwright.config.ts') }}
-          restore-keys: |
-            playwright-${{ runner.os }}-
-
-      - name: Install Playwright browsers
-        if: steps.playwright-cache.outputs.cache-hit != 'true'
-        run: pnpm exec playwright install --with-deps
-
-      - name: Install system dependencies for WebKit
-        # Some WebKit dependencies seem to lay outside the cache and will need to be installed separately
-        if: ${{ steps.playwright-cache.outputs.cache-hit == 'true' }}
-        run: pnpm exec playwright install-deps webkit
-
+      # This workflow is triggered for all deployments (liam-app, liam-erd-sample, liam-docs),
+      # but E2E tests are executed only for liam-app deployments now.
       - name: Run e2e tests
+        if: |
+          github.event_name == 'deployment_status' &&
+          github.event.deployment_status.state == 'success' &&
+          (
+            (
+              contains(github.event.deployment.environment, 'Production') &&
+              contains(github.event.deployment_status.target_url, 'liam-app-git-main')
+            ) ||
+            (
+              contains(github.event.deployment.environment, 'Preview') &&
+              !contains(github.event.deployment_status.target_url, 'liam-erd-sample') &&
+              !contains(github.event.deployment_status.target_url, 'liam-docs')
+            )
+          )
         run: pnpm exec playwright test --project="${{ matrix.browser }}"
         env:
           URL: ${{ env.URL }}
-
-      - name: Upload test results
-        uses: actions/upload-artifact@v4
-        if: failure()
-        with:
-          name: test-results-${{ matrix.browser }}
-          path: frontend/packages/e2e/test-results/
-          retention-days: 30
-
-      - name: Slack Notification on Failure
-        if: ${{ env.ENVIRONMENT == 'Production – liam-app' && failure() }}
-        uses: tokorom/action-slack-incoming-webhook@d57bf1eb618f3dae9509afefa70d5774ad3d42bf # v1.3.0
-        env:
-          INCOMING_WEBHOOK_URL: ${{ secrets.SLACK_CLI_CI_WEBHOOK_URL }}
-        with:
-          text: "E2E Test Failure"
-          attachments: |
-            [
-              {
-                "color": "bad",
-                "fields": [
-                  {
-                    "title": "Browser",
-                    "value": "${{ matrix.browser }}"
-                  },
-                  {
-                    "title": "Result",
-                    "value": "failure"
-                  },
-                  {
-                    "title": "Job URL",
-                    "value": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-                  }
-                ]
-              }
-            ]
```

Comment:
Q: Is it intentional to delete a notification?

Reply:
Oh, I have added unnecessary correspondence.🙏
This slack notification, however, is currently not working, so I will undo the changes and only add a comment.

Reply:
@FunamaYukina I see, It became to skip notification even when failing 💭 I wonder when started.

<img width="878" alt="Screenshot_2025-04-01_at_11_34_44" src="https://github.com/user-attachments/assets/b165f112-0e2a-460f-8444-5f43866b2424" />

ref: https://github.com/liam-hq/liam/actions/runs/14169776317/job/39690805919

I think this PR is good to merge as is. But We want to fix it!

Reply:
Even when merged into the main branch, it seemed to be skipped because the environment would be ``preview``.
https://github.com/liam-hq/liam/actions/runs/14129036651/job/39584629358
![ss 3016](https://github.com/user-attachments/assets/a0881d51-83e4-4439-8f9d-157cb46d9597)

Yes. We would like to correct that.🔧

Reply:
I have put the code back in place!
https://github.com/liam-hq/liam/pull/1053/commits/d25ed7e25baea87054cd807f184d82477ebf3f4d